Traceback (most recent call last):
  File "trainer_contrastive_learning.py", line 347, in <module>
    main(opt)
  File "trainer_contrastive_learning.py", line 332, in main
    dataset_dict = load_split_dataset(cfg)
  File "trainer_contrastive_learning.py", line 144, in load_split_dataset
    pair_set_dict = create_split_pair_set(cfg, data_path=cfg.seq_data)
  File "trainer_contrastive_learning.py", line 100, in create_split_pair_set
    df = pd.read_csv(data_path, index_col=0)
  File "/home/ksuga/whole_mrna_predictor/.venv/lib/python3.8/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/ksuga/whole_mrna_predictor/.venv/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/ksuga/whole_mrna_predictor/.venv/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 488, in _read
    return parser.read(nrows)
  File "/home/ksuga/whole_mrna_predictor/.venv/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1047, in read
    index, columns, col_dict = self._engine.read(nrows)
  File "/home/ksuga/whole_mrna_predictor/.venv/lib/python3.8/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 223, in read
    chunks = self._reader.read_low_memory(nrows)
  File "pandas/_libs/parsers.pyx", line 801, in pandas._libs.parsers.TextReader.read_low_memory
  File "pandas/_libs/parsers.pyx", line 857, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 843, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 1925, in pandas._libs.parsers.raise_parser_error
pandas.errors.ParserError: Error tokenizing data. C error: Calling read(nbytes) on source failed. Try engine='python'.